extensions/amp-inline-gallery/1.0/web-component.js

```diff
@@ -7,11 +7,17 @@ import {
   TAG as PAGINATION_TAG,
   PaginationBaseElement,
 } from './pagination-base-element';
+import {CSS as PAGINATION_CSS} from '#build/bento-inline-gallery-pagination-1.0.css.js';
+import {CSS} from '#build/bento-inline-gallery-1.0.css.js';
 
 /**
  * Registers `<bento-inline-gallery> component to CustomElements registry
  */
 export function defineElement() {
+  const style = document.createElement('style');
```

Comment:
Let's share a utility function to install CSS. It's okay to leave it in this file for now.